service/templates/admin.html

```diff
@@ -772,6 +773,24 @@ <h2 class="modal-title">[[modalTitle]]</h2>
             </md-card>
           </section>
 
+          <section layout="row" flex>
```

Comment:
The section above this one only has 1 card in it, so I would put this card in that section to keep the layout consistent

Reply:
My reasoning was that this panel is going to be disproportionately larger than its neighbors (we'll be configuring multiple container pools) so it might be best to start on a new row on the left-hand side.